plugin-management-library/src/test/java/io/jenkins/tools/pluginmanager/impl/PluginManager_appendPathOntoUrlTest.java

```diff
@@ -0,0 +1,73 @@
+package io.jenkins.tools.pluginmanager.impl;
+
+import io.jenkins.tools.pluginmanager.config.Config;
+import io.jenkins.tools.pluginmanager.config.Settings;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class PluginManager_appendPathOntoUrlTest {
+
+    @Test
+    public void appendPathOntoUrlTest() {
+        Config config = prepareMockConfig();
+        PluginManager objectUnderTest = new PluginManager(config);
+
+        String url = "http://bob.com:8080";
+        String path = "file.json";
+
+        String result = objectUnderTest.appendPathOntoUrl(url, path);
+
+        assertThat(result, is(url + "/" + path));
```

Comment:
can you not concat in the test it'll be much more readable if you just hard code it:
```suggestion
        assertThat(result, is("http://bob.com:8080/file.json"));
```

Reply:
I did it that way to highlight what the method being tested actually was expected to do. It added one slash between the two values being passed to it without changing them.  If you think its important, I don't mind repeating the actual text in the setup for the test and in the assertion.

Reply:
I think it’s worth doing, its much clearer to me

Reply:
Ok. Done.